app/views/organisations/index.html.erb

```diff
@@ -44,6 +44,13 @@
         <% end %>
       </tbody>
   </table>
+
+  <% if @services.count > 0 %>
```

Comment:
Is this meant to be `> 20`?

Reply:
wonder where that 2 went! 🤦‍♀️ 

Its now .length too!